requirements.json

```diff
@@ -175,14 +175,14 @@
         "name": "OpenJDK",
         "installable": true,
         "bundle": "yes",
-        "dmUrl": "https://developers.redhat.com/download-manager/jdf/file/java-1.8.0-openjdk-1.8.0.151-1.b12.redhat.windows.x86_64.msi?workflow=direct",
-        "url": "http://download.eng.bos.redhat.com/brewroot/packages/openjdk8-win/1.8.0.151/1.b12/win/java-1.8.0-openjdk-1.8.0.151-1.b12.redhat.windows.x86_64.msi",
-        "fileName": "java-1.8.0-openjdk-1.8.0.151-1.b12.redhat.windows.x86_64.msi",
-        "sha256sum": "6cc4b7b017cdd68f381184544743c17d2f84e6ff7cfc64575a20319b8b114e65",
-        "version": "1.8.0_151",
-        "size": 110055424,
-        "installSize": 213676032,
-        "virusTotalReport": "https://www.virustotal.com/#/file/6cc4b7b017cdd68f381184544743c17d2f84e6ff7cfc64575a20319b8b114e65/detection"
+        "dmUrl": "https://developers.redhat.com/download-manager/jdf/file/java-1.8.0-openjdk-1.8.0.161-1.b14.redhat.windows.x86_64.msi?workflow=direct",
+        "url": "https://developers.redhat.com/download-manager/jdf/file/java-1.8.0-openjdk-1.8.0.161-1.b14.redhat.windows.x86_64.msi?workflow=direct",
```

Comment:
Use internal URL for url attribute 
http://download.eng.bos.redhat.com/brewroot/packages/openjdk8-win/1.8.0.161/1.b14/win/java-1.8.0-openjdk-1.8.0.161-1.b14.redhat.windows.x86_64.msi
It is used during the build only for bundled components.